diff --git a/node_modules/react-native-floating-label-input/src/index.tsx b/node_modules/react-native-floating-label-input/src/index.tsx
index 439fadb..88998ef 100644
--- a/node_modules/react-native-floating-label-input/src/index.tsx
+++ b/node_modules/react-native-floating-label-input/src/index.tsx
@@ -592,6 +592,7 @@ const FloatingLabelInput: React.ForwardRefRenderFunction<InputRef, Props> = (
 
   return (
     <TouchableWithoutFeedback
+      accessible={false}
       style={{ flex: 1 }}
       onPress={setFocus}
       onLayout={onLayout}
